morphir/runtime/test/jvm/src/org/finos/morphir/runtime/EvaluatorMDMTests.scala

```diff
@@ -3448,6 +3469,57 @@ object EvaluatorMDMTests extends MorphirBaseSpec {
             )
           )
         )
+      ),
+      suite("Morphir Runtime Exception Tests")(
+        testExceptionMultiple("FailedCoercion Test")(
+          "exceptionTests",
+          "sdkAddTest",
+          List(Data.String("a"), Data.String("b"))
+        ) {
+          case TopLevelError(_, _, _) => assertTrue(true)
+          case e                      => assertNever(s"Expected FailedCoercion but $e was thrown")
+        },
+        testExceptionMultiple("UnknownTypeMismatch Test")("exceptionTests", "decimalHundred", List(Data.String("a"))) {
+          case TopLevelError(_, _, _: TypeError.UnknownTypeMismatch) => assertTrue(true)
+          case TopLevelError(_, _, inner) => assertNever(s"Expected UnsupportedType but $inner was thrown")
```

Comment:
I think these "Expected" look wrong - shouldn't this be `UnknownTypeMismatch` not `UnsupportedTypeMismatch`? Given the brittleness I would not specify the expectation at all ("Unexpected exception type thrown")

Reply:
Fair, will update